txpool/pool.go

```diff
@@ -1701,7 +1703,7 @@ func (p *TxPool) fromDB(ctx context.Context, tx kv.Tx, coreTx kv.Tx) error {
 		if err != nil {
 			return err
 		}
-		addr, txRlp := v[:20], v[20:]
+		addr, txRlp := *(*[20]byte)(v[:20]), v[20:]
```

Comment:
let's use `BytesToAddress` func 
and maybe change implementation of `BytesToAddress` to use this "casting trick".

Reply:
Good advice, will try to implement